modules/cql/src/blaze/elm/compiler/type_operators.clj

```diff
@@ -228,7 +228,65 @@
     (->DescendentsOperatorExpression source)))
 
 
-;; TODO 22.18. Is
+;; 22.18. Is
+(defrecord IsExpression [operand type pred]
+  core/Expression
+  (-eval [_ context resource scope]
+    (pred (core/-eval operand context resource scope)))
```

Comment:
I made sure, that we can just call `prep` here without doing anything additionally because of performance.